### PR TITLE
Configure grunt to replace require.js with almond during build

### PR DESCRIPTION
**Pull request by stytz**
Desided to push as a separate feature, conflicts may occur if merged with pull-request #3.

### CODE REVIEW

**Comment by stytz** [comment]
If both pull-requests #3 and #4 are merged, **index.html** should contain just this:

``` html
<script data-main="js/init.js" src="js/bower/requirejs/require.js"></script>
```

File: Gruntfile.js [hunk 1]
```diff
@@ -43,11 +43,17 @@ module.exports = function (grunt) {
         requirejs: {
             dist: {
                 options: {
-                    optimize: 'none',
+                    almond: true,
+                    insertRequire: ['main'],
+                    replaceRequireScript: [{
+                        files: ['<%= yeoman.dist %>/index.html'],
+                        // module: 'main',
```
Comment (TanyaGaleyev) [inline Gruntfile.js:50]: do we need this comment?

File: app/index.html [hunk 2]
```diff
@@ -61,10 +61,9 @@
 <script src="js/inc/shim/Base64.js" type="text/javascript"></script>
 <script src="js/inc/shim/Base64binary.js" type="text/javascript"></script>
 <script src="js/inc/shim/WebAudioAPI.js" type="text/javascript"></script>
-<script src="js/bower/requirejs/require.js"></script>
 <script src="js/inc/WebMIDIAPI.min.js"></script>
-<!-- build:js({.tmp}) js/main.js -->
-<script src="js/init.js"></script>
-<!-- endbuild -->
+
+<script data-main="js/init.js" src="js/bower/requirejs/require.js"></script>
```
Comment (TanyaGaleyev) [inline app/index.html:61]: omfg, data-main, see it first time =)

Comment (stytz) [inline app/index.html:61]: It's _require.js_-only attribute =) one of the possible ways to specify main script entry point. And here it's necessary cause grunt-requirejs **almondify-feature** uses it to find and replace script tag.

File: Gruntfile.js (hunk 1 above)
Comment (stytz) [inline Gruntfile.js:50]: _//module: 'main'_ ? You can remove it )

**Pull request merged**